client/src/components/Dashboard/index.js

```diff
@@ -45,21 +181,60 @@ class Dashboard extends Component {
               >
                 <Grid item xs={12} md={12}>
                   <div className={classes.liveticker}>
-                      <KeyIndicators />
+                    <KeyIndicators />
                   </div>
                 </Grid>
 
                 <Grid item xs={12} md={12}>
                   <Paper className={classes.paper}>
-                      <YourFavorites />
+                    <YourFavorites />
                   </Paper>
                 </Grid>
                 <Grid item xs={12}>
                   <div className={classes.block}>
-                    <Typography variant="h5" gutterBottom style={{paddingLeft:"20px"}}>
+                    <Typography variant="h5" gutterBottom>
                       Dashboard
                     </Typography>
-                      <LiveTicker />
+                    <div>
+                      <Autosuggest
```

Comment:
I don't think the search bar should extend all the way across—maybe just a little bit longer than the placeholder text.